include/psa/crypto_platform.h

```diff
@@ -44,57 +44,33 @@
 /* PSA requires several types which C99 provides in stdint.h. */
 #include <stdint.h>
 
+#if ( defined(__ARMCC_VERSION) || defined(_MSC_VER) ) && \
+    !defined(inline) && !defined(__cplusplus)
+#define inline __inline
+#endif
+
```

Comment:
`crypto_struct.h` already contained `static line` functions and neither our CI nor any user have complained. Maybe it only got away with this accidentally because nobody has included it in a configuration that didn't also pull in `mbedtls/cipher.h` or `mbedtls/pk.h`. But in principle, this should no longer be necessary since we require C99. So please don't add it.

Reply:
I added this due to a build falling in the CI. As things have changes quite significantly let's remove it and see if the CI still complains.

Reply:
Some Windows builds fail without this thus I've added it back.